src/OpenDiffix.Core/Analyzer.fs

```diff
@@ -315,6 +315,9 @@ let private addLowCountFilter aidColumnsExpression selectQuery =
       |> List.map (fun selectedColumn -> selectedColumn.Expression)
 
     if selectedExpressions |> List.forall Expression.isScalar then
+      // No need to group implicitly by constants, which are invalid bucket definitions anyway.
+      let implicitGroupExpressions = selectedExpressions |> List.filter (Expression.isConstant >> not)
```

Comment:
Hm, if I do that, it fails one of the tests. I'll investigate why.

Reply:
the test failed, because it compared SQL layer seed and https://github.com/diffix/reference/blob/master/src/OpenDiffix.Core.Tests/Analyzer.Tests.fs#L306-L311 didn't have `Normalizer.normalize` in the pipeline. But I think it should, why would we want it to be the odd one out?

If I put `Normalizer.normalize` in, there's another test failing, one about consistency of seeding numeric ranges. If the `normalize` should indeed be there like in production code, then this indicates we have a bug, I'll investigate

Reply:
For future reference: After some discussion elsewhere, we decided to just comment out the offending test cases, and fix in a follow-up PR. The fix I proposed which didn't get into `master` is in 29da4c72ef7.

Reply:
And the fix for the original request in this thread is 593f3e2d, LMK if this is what you've meant